superset-frontend/src/dashboard/components/dnd/handleHover.js

```diff
@@ -18,6 +18,7 @@
  */
 import { throttle } from 'lodash';
 import getDropPosition from '../../util/getDropPosition';
+import scrollDashboardTop from './handleScroll';
```

Comment:
Can we make the default export to have the same name as the filename, or do not use export default?

You can temporarily disable the `prefer-default` lint rule if that is what's been blocking you from not using default.